### DISCUSSION TITLE
Removing logs from console

### DISCUSSION THREAD

**Discussion by teverus** [webfp/tor-browser-selenium#149] [General]
Hello! I have a quick question: is there a way to disable logs in the console? It's just when I start some logs appear in the console and I want them to go away :( Is there a way to do it? :) Thank you! :)
**UPDATE**: I noticed that when I start a new project, the amount of logs is even bigger. Could you please tell me if I can disabled them?

![Image 146](https://user-images.githubusercontent.com/62369371/116804132-13d81500-ab25-11eb-98e3-084196a96943.png)
df

**Comment by gunesacar** [comment]
Hello @teverus,

Thanks for your question.

I believe these log lines originate from geckodriver, but you may be able to disable them by passing `/dev/null` in [`tbb_logfile_path` parameter during initialization](https://github.com/webfp/tor-browser-selenium/blob/main/tbselenium/tbdriver.py#L35). Would you mind giving that a try and sharing the result?

**Reply by teverus** [reply]
Thank you so much, it worked! :) I redirected the output to a file that I don't really care about; so, the console is clean now :) Thank you once again :)
![Image 151](https://user-images.githubusercontent.com/62369371/116862286-35acc700-ac0d-11eb-876b-0022ec80a583.png)

**Reply by gunesacar** [reply]
Thanks. Good to know that it worked.

As a side note, I'd avoid sharing blurred screenshots of potentially sensitive data.
See, https://hovav.net/ucsd/dist/redaction.pdf

**Comment by dongu54321** [comment]
Hello @teverus, how do you make it work for windows?

**Reply by teverus** [reply]
I just added .exe to this contant and that's it
![Image 1656](https://user-images.githubusercontent.com/62369371/117412195-2b484100-af1d-11eb-89f8-09fe68126854.png)

**Reply by dongu54321** [reply]
thanks @teverus
